tools/packaging/kata-deploy/README.md

```diff
@@ -66,42 +76,49 @@ spec:
 To run an example with `kata-clh`:
 
 ```sh
-$ cd $GOPATH/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/examples
-$ kubectl apply -f test-deploy-kata-clh.yaml
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-clh.yaml
 ```
 
 To run an example with `kata-fc`:
 
 ```sh
-$ cd $GOPATH/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/examples
-$ kubectl apply -f test-deploy-kata-fc.yaml
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-fc.yaml
 ```
 
 To run an example with `kata-qemu`:
 
 ```sh
-$ cd $GOPATH/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/examples
-$ kubectl apply -f test-deploy-kata-qemu.yaml
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-qemu.yaml
 ```
 
 The following removes the test pods:
 
 ```sh
-$ cd $GOPATH/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/examples
-$ kubectl delete -f test-deploy-kata-clh.yaml
-$ kubectl delete -f test-deploy-kata-fc.yaml
-$ kubectl delete -f test-deploy-kata-qemu.yaml
+$ kubectl delete -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-clh.yaml
+$ kubectl delete -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-fc.yaml
+$ kubectl delete -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/examples/test-deploy-kata-qemu.yaml
 ```
 
 ### Remove Kata from the Kubernetes cluster
 
+#### Removing the latest image
+
+```sh
+$ kubectl delete -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml
+$ kubectl delete -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml
```

Comment:
Does it work to run these commands all at once?  Just looking at it it seems you have to let the `kata-cleanup.yaml` pod enough time to start and finish doing its thing before you delete the pod.

Reply:
You're right, I'll change all the links.  And it makes me think what I can do with the `k3s` one.

Reply:
While I agree, I didn't change this in this PR.  Anyways, we can work this around by adding a wait condition.

Reply:
@eadamsintel, would you mind if I open a new issue and address this one in a different series?

I think that what we can do to ensure that kata-deploy was properly deployed is:
```sh
$ kubectl -n kube-system wait --for=condition=Ready pod `kubectl -n kube-system get pods -o'custom-columns=name:metadata.name' --no-headers | grep kata-deploy-`
```

Then to ensure it get deleted:
```sh
kubectl -n kube-system wait --for=delete pod `kubectl -n kube-system get pods -o'custom-columns=name:metadata.name' --no-headers | grep kata-deploy-`
```

And similar things for the other calls, but I am pretty sure there's a better way to do that and maybe it's not worth rushing it into the release considering the instructions were already there before for a long time.

WDYT?

The other two points you raised I'll have them addressed by the end of the day.

Reply:
You could also just break up the code block after the cleanup pod is deployed. You could then always just say in text "Wait for the `kata-deploy-cleanup` pod to complete before running the next commands.  That command above wouldn't fit in nicely with the other commands.